addons/im_livechat/report/im_livechat_report_channel.py

```diff
@@ -18,10 +18,23 @@ class ImLivechatReportChannel(models.Model):
     technical_name = fields.Char('Code', readonly=True)
     livechat_channel_id = fields.Many2one('im_livechat.channel', 'Channel', readonly=True)
     start_date = fields.Datetime('Start Date of session', readonly=True, help="Start date of the conversation")
+    start_hour = fields.Char('Start Hour of session', readonly=True, help="Start hour of the conversation")
+    day_number = fields.Char('Day Number', readonly=True, help="Day number of the session (1 is Monday, 7 is Sunday)")
+    time_to_answer = fields.Float('Time to answer (sec)', digits=(16, 2), readonly=True, group_operator="avg", help="Average time in seconds to give the first answer to the visitor")
     start_date_hour = fields.Char('Hour of start Date of session', readonly=True)
     duration = fields.Float('Average duration', digits=(16, 2), readonly=True, group_operator="avg", help="Duration of the conversation (in seconds)")
     nbr_speaker = fields.Integer('# of speakers', readonly=True, group_operator="avg", help="Number of different speakers")
     nbr_message = fields.Integer('Average message', readonly=True, group_operator="avg", help="Number of message in the conversation")
+    is_without_answer = fields.Integer('Session(s) without answer', reandonly=True, group_operator="sum",
+                                       help="""A session is without answer if the operator did not answer. 
+                                       If the visitor is also the operator, the session will always be answered.""")
+    days_of_activity = fields.Integer('Days of activity', group_operator="max", readonly=True)
+    is_anonymous = fields.Integer('Is visitor anonymous', readonly=True)
+    country = fields.Char('Country of the visitor', readonly=True)
+    is_happy = fields.Integer('Visitor is Happy', readonly=True)
+    rating = fields.Integer('Rating', group_operator="avg", readonly=True)
```

Comment:
`rating_satisfaction`?  Can't it be computed as the `AVG(Rating.rating)` and avoid to put it in the group by clause ?